docs/source/dev/tests.md

```diff
@@ -58,6 +58,8 @@ pnpm --prefix ui build
 CYPRESS_BASE_URL="http://127.0.0.1:8081" pnpm --prefix ui e2e
 ```
 
+> Currently, WebSockets don't work when running Cypress in the CI. In order to replicate the CI environement more accurately, WebSockets can be disabled locally via environment variable `VITE_DISABLE_WEBSOCKETS`. For instance, you can start the development server with sockets disabled with: `VITE_DISABLE_WEBSOCKETS=true pnpm --prefix ui start`.
```

Comment:
Running Cypress locally was still broken. I introduced that env var `VITE_DISABLE_WEBSOCKETS` a few weeks back, but I was setting it in the `cypress` script in `ui/package.json`. This had no effect since the env var is used by the front-end code to decide not to open socket connections -- not by Cypress.

Obviously we don't want to always start the dev server with sockets disabled, so I decided the simplest was to document the command `VITE_DISABLE_WEBSOCKETS=true pnpm --prefix ui start` for now. 

To summarize, to run E2E tests locally against the front-end development server, run the following three commands in separate terminals:

- `mxcubeweb-server -r $(pwd)/demo/ --static-folder $(pwd)/ui/build/ -L debug`
- `VITE_DISABLE_WEBSOCKETS=true pnpm --prefix ui start`
- `pnpm --prefix ui cypress`

Reply:
I think we should add `127.0.0.1:8081` to he CORS hosts as well since we are doing `CYPRESS_BASE_URL: http://127.0.0.1:8081` in the CI actions.

Reply:
Just a thought, even if its not directly related :)

Reply:
You're a genius!

![image](https://github.com/user-attachments/assets/1040404f-0bac-4858-be9d-b796661d4c67)

Reply:
;), not sure but I sometimes manage to get things right